.happy/config.json

```diff
@@ -80,5 +80,8 @@
   "tasks": {
     "migrate": ["migrate_db_task_definition_arn"],
     "delete": ["delete_db_task_definition_arn"]
+  },
+  "features": {
+    "enable_dynamo_locking": true
```

Comment:
The PR states that this is intended to solve the problem of not correctly cleaning up ECS tasks when tearing down an rdev instance. But what does this enabling this attribute actually do? What's the mechanism?

Reply:
This locks the db that stores information about each happy environment when someone is writing or deleting. This is to prevent a race condition which can causes the db to out of sync with reality.

Reply:
Good to know